project_status/models/__init__.py

```
@@ -0,0 +1,2 @@
+from . import project
```

Ruff (0.11.9)

`.project` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)